docs/report.md

```diff
@@ -1,36 +1,72 @@
-# Example Device Test Report
+# DAQ scan report for device 9a02571e8f00
```

Comment:
This isn't right -- this is an example, that's what the title is about -- why is this all changing?

Reply:
"an example" of how to view the test result output, not an example of the test output itself.  The key part of this file is the "cat" command, which you're removing!

Reply:
OK, understood. I totally missed it was an example! I'll regenerate and make sure it is "an example".

Reply:
Did you update this, still doesn't seem right.

Reply:
Will be updated in next commit, sorry for missing this.

Reply:
Now updated.